extensions.c

```diff
@@ -774,14 +778,15 @@ hibaext_sanity_check(const struct hibaext *ext) {
 							ret = HIBA_GRANT_BADOPTIONS;
 						break;
 					case '\'':
```

Comment:
Do we care about single quotes? The options quoting in OpenSSH for authorized_keys is quite strict. It only allows double-quoting and only supports escaping double quotes. See `opt_dequote` function in `misc.c` and the callers in `auth-options.c`.

Reply:
Good point, I remove the escaping check of single quotes.

Reply:
I think we can remove the state tracking for single quotes altogether, since OpenSSH only allows double quoting for the option values.